requirements.txt

```diff
@@ -1,8 +1,7 @@
 # TODO: put back the standard pypi release when
 # next point version of openwisp-controller is released
 openwisp-controller @ https://github.com/openwisp/openwisp-controller/tarball/master
-# openwisp-utils[rest] is left unpinned so the version used
-# is the one dictated by openwisp-controller
+openwisp-utils[rest] @ https://github.com/openwisp/openwisp-utils/tarball/master
```

Comment:
this shouldn't be needed: https://github.com/openwisp/openwisp-controller/blob/master/requirements.txt#L10

Reply:
removed!